packages/dao/contracts/Api3BaseTemplate.sol

```diff
@@ -112,20 +112,22 @@ contract Api3BaseTemplate is IsContract {
 
     /* AGENT */
 
-    function _installDefaultAgentApp(Kernel _dao) internal returns (Agent) {
+    function _installDefaultAgentApps(Kernel _dao) internal returns (Agent, Agent) {
         bytes memory initializeData = abi.encodeWithSelector(Agent(0).initialize.selector);
-        Agent agent = Agent(_installDefaultApp(_dao, AGENT_APP_ID, initializeData));
+        Agent mainAgent = Agent(_installDefaultApp(_dao, AGENT_APP_ID, initializeData));
+        Agent secondaryAgent = Agent(_installDefaultApp(_dao, AGENT_APP_ID, initializeData));
```

Comment:
One of the two Agent app - probably the less secure one - should be installed using ` _installNonDefaultApp` as defined [here]( https://github.com/aragon/dao-templates/blob/3e1c3a5f986840d7380015a0c7405d4cdf73563e/shared/contracts/BaseTemplate.sol#L340). Otherwise the OS won't know which one is the default one - and thus which one is the recovery vault.

Reply:
Actually that's a good point, I've noticed that we don't need to change Api3BaseTemplate at all actually..